api/src/test/scala/org/ensime/api/EnsimeTestData.scala

```diff
@@ -14,7 +14,7 @@ trait EnsimeTestData {
     }
   }
 
-  val typeInfo = new BasicTypeInfo("type1", DeclaredAs.Method, "FOO.type1", List(), List(), None)
+  val typeInfo = BasicTypeInfo("type1", DeclaredAs.Method, "FOO.type1", Nil, Nil, None, Nil)
```

Comment:
we use `new` in a whole load of places, it's because of the path that the code has taken... it wasn't always as clean an ADT as it is today and used to be combined with the modelbuilder file.